### PR TITLE
Create STYLE_GUIDE.md

### PR DESCRIPTION
**Pull request by Aichnerc**
The styleguide has been implemented.

**Checklist:**

 
 - [ ] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [x] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no styleguide.

**What is the new behavior (if this is a feature change)?**
- The styleguide has been added.

**Other information**:
- Ref: https://github.com/snek-at/tonic/edit/master/STYLE_GUIDE.md

### CODE REVIEW

File: STYLE_GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+
+It is recommended to use Visual Studio Code with all recommended Extensions when working on code
+with the agency or under the agency’s supervision. Use the built-in code quality
+improvement tool `Prettier` as often as possible. It does not only help you to keep
+focus and be more efficient, it also improves the code quality tremendously.
+
+All HTML and CSS should conform to the [Code Guide](https://github.com/mdo/code-guide), maintained by [Mark Otto](https://github.com/mdo).
+
+The following rules are one of the most important.
+- Try to keep the maximum line length below `80`. The maximum line length is `120`.
```
Comment (Theralun) [inline STYLE_GUIDE.md]: I'd get rid of the first `maximum`

**Review by Theralun — CHANGES_REQUESTED** [review]
Should be good after a minor change

**Review by LucaAllmaier — APPROVED** [review]
Looks good.

**Review by Theralun — APPROVED** [review]
looks good.

**Pull request merged**